### ISSUE TITLE
Availability of Immunocto-pretrained model / training script

### ISSUE THREAD

**Issue by titusgriebel** [mikaelsimard5/Immunocto#2]
Hi, thank you so much for this large high-quality dataset!
For my use, access to the checkpoint trained on the Immunocto database would be very helpful.
Is it possible for you to make it available, potentially also the code to train it from Immunocto V1? I would like to see how training works without lymphocyte subtyping (not required for my purposes) and potentially leverage the valuable lymphocyte labels for downstream tasks.

Thanks!
